source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -147,7 +147,7 @@ public PimsProperty GetById(long id)
         /// </summary>
         /// <param name="ids"></param>
         /// <returns></returns>
-        public PimsProperty GetAllByIds(IEnumerable<long> ids)
```

Comment:
Changing the return type makes sense, but why the input type? are we saying only lists are valid inputs but not arrays, sets, etc?

Reply:
We can change it if the need arise. As it is it, only passes things from the controller, which only takes a list